src/routes/process-user-vocabulary/components/ImageSelectionOverlay.tsx

```diff
@@ -25,37 +25,31 @@ const ActionBar = styled.View`
 const TakeImageButton = styled.Pressable`
   flex: 2;
   align-items: center;
-  margin-right: ${GALLERY_IMAGE_SIZE}px;
+  margin-right: ${GALLERY_ICON_SIZE}px;
 `
 
 const Container = styled.View`
   flex-direction: row;
   justify-content: center;
 `
 
-interface Props {
+interface ImageSelectionOverlayProps {
   setVisible: (visible: boolean) => void
   numberOfImages: number
   pushImage: (imageUri: string) => void
 }
 
-// TODO
-// adjust and test on ios
-// write test
-
-const ImageSelectionOverlay = ({ setVisible, pushImage, numberOfImages }: Props): ReactElement => {
+const ImageSelectionOverlay = ({ setVisible, pushImage, numberOfImages }: ImageSelectionOverlayProps): ReactElement => {
   const camera = useRef<RNCamera>(null)
 
-  const takePicture = async () => {
+  const takeImage = async () => {
     if (camera.current) {
       const options = { quality: 0.5, base64: true }
       const data = await camera.current.takePictureAsync(options)
+      setVisible(false)
       if (numberOfImages < 3) {
         pushImage(data.uri)
       }
```

Comment:
This should not be needed anymore, right?
```suggestion
      pushImage(data.uri)
```